twisted/python/dist3.py

```diff
@@ -70,6 +70,7 @@
     "twisted.conch.ssh.transport",
     "twisted.conch.ssh.channel",
     "twisted.conch.ssh.userauth",
+    "twisted.conch.ssh.session",
```

Comment:
This is OK, but as you are adding modules to dist3,
try to add them so that they are sorted.